components/icu/Cargo.toml

```diff
@@ -39,6 +39,7 @@ icu_timezone = { workspace = true }
 icu_compactdecimal = { workspace = true, optional = true }
 icu_displaynames = { workspace = true, optional = true }
 icu_relativetime = { workspace = true, optional = true }
+icu_transliterate = { workspace = true, features = ["compile"], optional = true } # currently only usable with the compile feature
```

Comment:
Suggestion: add the "compile" feature to the default features of the icu metacrate